pkgs/development/python-modules/pyparser/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildPythonPackage, fetchPypi, fetchpatch, python, stdenv, pythonPackages }:
+
+let
+  # pyparser only builds with parse == 1.6.5
+  parse = buildPythonPackage rec {
```

Comment:
We can't have "private" packages inside `python-packages.nix` as that could cause collisions.

Reply:
So what do you suggest?